### PR TITLE
upgrade dock sdk

### PR DESCRIPTION
**Pull request by maycon-mello**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,13 +17,13 @@
     "@polkadot/types-known": "5.6.1",
     "@polkadot/util": "7.2.1",
     "@polkadot/rpc-core": "5.6.1",
```
Comment (SamHellawell) [inline package.json:19]: should update all polkadot resolutions to match SDK's, or try remove them

File: package.json [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
   },
   "devDependencies": {
     "@babel/core": "^7.16.0",
     "@babel/preset-env": "^7.14.5",
-    "@docknetwork/sdk": "1.2.3",
-    "@docknetwork/wallet": "0.0.3",
+    "@docknetwork/sdk": "2.2.0",
+    "@docknetwork/wallet": "0.2.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.4.3",
     "@polkadot/keyring": "^7.2.1",
```
Comment (SamHellawell) [inline package.json:28]: similar here ensure polkadot versions are upto date with what SDK uses

File: package.json (hunk 1 above)
Comment (maycon-mello) [reply package.json:19]: Prob thats why build is failing, will check this out thanks

File: package.json (hunk 2 above)
Comment (SamHellawell) [reply package.json:28]:
```
    "@polkadot/api": "7.15.1",
    "@polkadot/keyring": "8.7.1",
    "@polkadot/rpc-provider": "7.15.1",
    "@polkadot/types": "7.15.1",
    "@polkadot/types-known": "7.15.1",
    "@polkadot/util": "8.7.1",
    "@polkadot/util-crypto": "8.7.1"
```

just for ref

File: package.json (hunk 1 above)
Comment (SamHellawell) [reply package.json:19]: yeah the "/sr25519" not exported error is related to it pulling an older polkadot version

**Review by SamHellawell — APPROVED** [review]

**Pull request merged**